src/pages/404.astro

```diff
@@ -0,0 +1,22 @@
+---
+import Layout from "../layouts/Layout.astro";
+import { Image } from "astro:assets";
+import Error from "../images/error.png";
+
+
+const description =
+  "The page you are looking for could not be found. Please check the URL and try again.";
+---
+
+<Layout title="404: Not Found - FujoCoded LLC" description={description}>
+  <article class="rounded-xl bg-white drop-shadow-purple">
+    <Image src={Error} alt="Boba-tan in her fursuit crawling around looking for her glasses" />
+    <h1><em>Oops!</em> This page could not be found.</h1>
```

Comment:
Can you add a button that says "take me home" and links to the homepage?

Maybe we should also add the newsletter signup/socials. "You haven't found us at this page, but you can still find us here". Or something corny.

Reply:
Sounds good but uh, I can't for the life of me find the link to the newsletter lmao